lib/matplotlib/pyplot.py

```diff
@@ -289,9 +298,13 @@ class backend_mod(matplotlib.backend_bases._Backend):
     # Need to keep a global reference to the backend for compatibility reasons.
     # See https://github.com/matplotlib/matplotlib/issues/6092
     matplotlib.backends.backend = newbackend
+    switch_backend._ever_called = True
+switch_backend._ever_called = False
```

Comment:
Instead of a separate flag, I guess you can globally init _backend_mod to None and test if `_backend_mod is None`?  (This seems clearer to me because the effect of switch_backend is to set _backend_mod to its correct, non-None value.)

Reply:
Good points.